1.0/docs/devguide/utility-functions.md

```diff
@@ -54,13 +54,13 @@ provides a set of useful convenience functions for instances to use.
 
 *   `isDebouncerActive(jobName)`. Returns true if the named debounce task is waiting to run.
 
-*   `transform(transform, node)`. Applies a CSS transform to the specified node,
-    or this element if no node is specified.
+*   `transform(transform, [node])`. Applies a CSS transform to the specified node,
```

Comment:
nvm on the `[]`. I see we're already using it on this page and we should be consistent with it.